xml/System.Diagnostics/Activity.xml

```diff
@@ -320,8 +321,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>A substring from <see cref="P:System.Diagnostics.Activity.Id" /> (or <see cref="P:System.Diagnostics.Activity.ParentId" />) between '|' (or beginning) and first '.'. Filtering by root Id allows to find all Activities involved in operation processing. RootId may be <see langword="null" /> if <see cref="T:System.Diagnostics.Activity" /> has neither <see cref="P:System.Diagnostics.Activity.ParentId" /> nor <see cref="P:System.Diagnostics.Activity.Id" />. See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#id-format" /> for more details.</summary>
```

Comment:
All instances of `ID` should be capitalized

Reply:
> Filtering by root Id allows to find all Activities involved in operation processing

Should be:

> Filtering by root Id allows for finding all Activities involved in operation processing

Reply:
```suggestion
        <summary>A substring from <see cref="P:System.Diagnostics.Activity.Id" /> (or <see cref="P:System.Diagnostics.Activity.ParentId" />) between '|' (or beginning) and first '.'.
Filtering by RootId allows to find all Activities involved in operation processing.
The RootId may be <see langword="null" /> if the <see cref="T:System.Diagnostics.Activity" /> has neither a <see cref="P:System.Diagnostics.Activity.ParentId" /> nor an <see cref="P:System.Diagnostics.Activity.Id" />.
See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#id-format" /> for more details.</summary>
```